internal/route/request_executor.go

```diff
@@ -50,6 +49,89 @@ func (r *RequestExecutor) routeToConfig(
 		httpReq.Header.Set("Authorization", "Basic "+encodedCredentials)
 	}
 
+	var (
+		respBody jsonrpc.ResponseBody
+		resp     *http.Response
+	)
+
+	_, isSingleRequestBody := requestBody.(*jsonrpc.SingleRequestBody)
+
+	if r.cache != nil && r.cache.ShouldCacheMethod(requestBody.GetMethod()) && isSingleRequestBody {
```

Comment:
Should we measure the latency of the cache?

Reply:
Should we emit metrics on cache hit ratio? One spitball idea is to have a counter for cache hits and cache misses.

Reply:
We have metrics coming from AWS that have this type of information.  I'd rather see if we can use those first before writing custom metrics TBH.